include/openthread-types.h

```diff
@@ -175,6 +175,25 @@ typedef struct otMeshLocalPrefix
     uint8_t m8[OT_MESH_LOCAL_PREFIX_SIZE];
 } otMeshLocalPrefix;
 
+#define OT_CHANNEL_MASK_ENTRIES_MAX_NUM    2   ///< Maximum number of Channel Mask Entries
+#define OT_CHANNEL_MASK_MAX_SIZE           4   ///< Maximum size of Channel Mask
+
+/**
+  * This structure represents Channel Mask Entry.
+  *
+  */
+typedef struct otChannelMaskEntry
+{
+    uint8_t mChannelPage;
+    uint8_t mMaskLength;
+    uint8_t m8[OT_CHANNEL_MASK_MAX_SIZE];
+} otChannelMaskEntry;
+
+typedef struct otChannelMask
+{
+    otChannelMaskEntry entries[OT_CHANNEL_MASK_ENTRIES_MAX_NUM];
```

Comment:
@jwhui ,I am trying to simply the API and do not add a new API for setting TLV data payload.
But seems that I need to modify the data structure of ChannelMaskTlv, ChannelMaskEntry, if I want to keep Set(tlv) consistent with others.
These 2 data structures are to be designed for multiple entries. But seems that it is not a good idea to change this.